e2e/interchaintestv8/cosmos_relayer_test.go

```diff
@@ -488,5 +515,34 @@ func (s *CosmosRelayerTestSuite) ICS20TimeoutPacketTest(ctx context.Context, num
 			s.Require().NotNil(resp.Balance)
 			s.Require().Equal(testvalues.InitialBalance, resp.Balance.Amount.Int64())
 		}))
+
+		s.Require().True(s.Run("Verify balances on Chain B", func() {
+			resp, err := e2esuite.GRPCQuery[banktypes.QueryBalanceResponse](ctx, s.SimdB, &banktypes.QueryBalanceRequest{
+				Address: simdBUser.FormattedAddress(),
+				Denom:   s.SimdB.Config().Denom,
+			})
+			s.Require().NoError(err)
+			s.Require().NotNil(resp.Balance)
+			s.Require().Equal(originalBalance, resp.Balance)
+		}))
+	}))
+
+	s.Require().True(s.Run("Constructing relay packet after timeout should fail", func() {
+		resp, err := s.RelayerClient.RelayByTx(context.Background(), &relayertypes.RelayByTxRequest{
+			SrcChain:       s.SimdA.Config().ChainID,
+			DstChain:       s.SimdB.Config().ChainID,
+			TimeoutTxIds:   txHashes,
+			TargetClientId: ibctesting.FirstClientID,
+		})
+		s.Require().Error(err)
+		s.Require().Nil(resp)
+	}))
+
+	s.Require().True(s.Run("Receiving packets on Chain B after timeout should fail", func() {
+		resp, err := s.BroadcastSdkTxBodyGetResult(ctx, s.SimdB, s.SimdBSubmitter, 2_000_000, txBodyBz)
+		s.Require().Error(err)
+		// The pakcet receiving failed because of the timeout
+		s.Require().Contains(err.Error(), "timeout elapsed")
```

Comment:
Probably should use ErrorContains here or something of that sort. Something like:
```suggestion
		s.Require().ErrorContains(err, "timeout elapsed")
```

Reply:
resolved it myself